### ISSUE TITLE
Admin Page: no option to bulk activating / deactivating modules

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#4534]
In the current version of Jetpack, you can go to Jetpack > Settings in your dashboard and activate or deactivate all modules at once, thanks to the bulk editing options.

In Jetpack 4.3, you have 2 options:

1. You can disable JavaScript in your browser while visiting the new Jetpack dashboard, and the old Settings screen will appear.
2. You can go to the old settings URL if you have it bookmarked (`/wp-admin/admin.php?page=jetpack_modules`).

It would be nice to offer a quick access to those bulk editing options in the new Jetpack interface.

I can think of 2 ways to quickly add that option back, as a work-around:

1. Add a link to the old settings URL somewhere in the Jetpack dashboard; in the footer, for example.
2. Once the search feature will be available, include bulk editing capabilities to the search results page; one could search for "Appearance", and be able to activate / deactivate all Appearance related modules; A search for "all" could allow you to edit all modules at once.

**Comment by jeherve** [comment]
Thinking about this some more; if we can't implement this in 4.3, would it be possible to customize the existing search feature, and add "active" and "inactive" search keywords that would allow you to quickly see a list of active or inactive modules?

cc @dereksmart since you worked on Search.

**Comment by jessefriedman** [comment]
Would be good to investigate a new way to bulk change options by presenting the ability to turn on all "Engagement" or "Security" features, instead of all features at once, or cherry picking the ones you want. Which will allow us to continue the conversational flow around grouped features. 

thoughts @dereksmart @jeffgolenski @MichaelArestad ?

**Comment by MichaelArestad** [comment]
That will likely exist in the next iteration. :)

**Comment by jeffgolenski** [comment]
@jessefriedman  Definitely. And this goes waaaay back to some wireframes I created during our initial explorations late last year at the GM. Concepts are a little outdated in terms of UI, but still relevant. 
<img width="866" alt="screen shot 2016-08-31 at 4 27 32 pm" src="https://cloud.githubusercontent.com/assets/214813/18145117/cdd1a986-6f97-11e6-8834-50d4e972fb50.png">

**Comment by jessefriedman** [comment]
@jeffgolenski <3 it.  We could probably just get rid of the `Activate All` and `Deactivate All` buttons there. And just have the `Enable` button enable them all at once.

**Comment by jeherve** [comment]
Pasting remarks from #5121:

---------

There was nothing wrong with the 4.0 install UI. One list of all the modules that you could see at a glance they all had their own icons it was easy and efficient to look at what was installed and what wasn't. Sure I could check the search icon with isn't remotely user friendly even intuitive that you can click the search icon to see a list. 

![jetpack-settings](https://cloud.githubusercontent.com/assets/1909677/18324806/39e0b88c-74f3-11e6-83c8-f56a9ed663d1.png)

There is too much padding on the list. You have scroll to see what was is installed and absolutely no more bulk options. That is also completely horrible and a huge step backwards for Jetpack. You guys made major gains over the years with me and others improving how your plugin works making it easier and faster to check setting and move on and this just throws that all away.

![jetpack-settings-new](https://cloud.githubusercontent.com/assets/1909677/18325019/39203e9e-74f4-11e6-9678-c6eecf21d488.png)

For 99% of the install I create, 99% of the wordpress users I work, talk with, help, with want to see at a glance what the settings are. Most people are the set and forget it type and this completely new UI increase time lost to finding the setting. With the categories on the side you could segment the list to what you wanted to see. But This top menu UI is just horribly bad, space consuming time consuming, error prone (did I leave a module on another screen I forgot to check bad) 

I am sorry if I sound rude or ungrateful for the massive amount of work that goes into this plugin. But this truly a step backwards for usability and efficiency and makes me never want to install jetpack on another install.

Signed a very annoyed Robert Dall

**Comment by jeherve** [comment]
Also mentioned here:
https://wordpress.org/support/topic/new-stats-well-stink/page/2/#post-8157218

**Comment by georgestephanis** [comment]
Here's a itty bitty plugin that will add a "Jetpack > Modules" menu item that gives access to the legacy module management screen including bulk activate / deactivate.

https://github.com/georgestephanis/jetpack-module-list/blob/master/jetpack-module-list.php

**Comment by mrwweb** [comment]
The fact that clicking the search icons shows all the modules was completely lost on me until someone at JP support mentioned it. I know empty search results are starting to be a more common UI, but bulk managing modules settings feels way to import to hide like that. +1 for a link to "Bulk Management" or something like that.

**Comment by robertdall** [comment]
Even the word "list view" would have improved this. To suggest the search icon shows all the modules is akin to the hamburger menu in terms of usability.

**Comment by jeherve** [comment]
Also reported here:
https://wordpress.org/support/topic/where-did-omnisearch-stats-access-go/#post-8251469

> The old interface with the modules, and **especially the view of what’s active and what’s inactive**, was a lot better.

Emphasis is mine. I think that's something we'll need to address when introducing bulk activating / deactivating tools.

**Comment by jeherve** [comment]
Also requested here:
https://wordpress.org/support/topic/new-version-very-bad-interface/

**Comment by jeherve** [comment]
#5461

**Comment by flexseth** [comment]
@jeherve - feature is implemented, clearly available in the primary Jetpack menu!

https://yourwebsite.com/wp-admin/admin.php?page=jetpack_modules

<img width="1155" height="800" alt="Image" src="https://github.com/user-attachments/assets/fa103e78-2c23-4cd4-b218-d3e85cdeff91" />

**Comment by jeherve** [comment]
@flexseth Yes, but this issue is about the newer Jetpack settings interfaces. This is the old settings URL I mentioned in the issue above.

**Comment by flexseth** [comment]
@jeherve Ok, apologies, I saw 3 iterations of the bulk enabler, and it seemed like there was a new UI pushed that I had on my sites with "Modules" being on the My Jetpack sidebar links just last night. 

This issue is very stale, so I figured I would offer some info to help get it closed if it was not being worked on/finished.
